core/src/main/java/com/linkall/cdk/runtime/SinkWorker.java

```diff
@@ -1,9 +1,8 @@
-package com.linkall.cdk.runtime.http;
+package com.linkall.cdk.runtime;
```

Comment:
the sink is not implemented batch received

Reply:
tracked by https://github.com/linkall-labs/cdk-java/issues/77